server/src/api/router.go

```diff
@@ -147,9 +148,9 @@ func (s *Server) protectedRoutes(r chi.Router) {
 			r.With(s.BoardParticipantContext).Get("/export", s.exportBoard)
 			r.With(s.BoardParticipantContext).Post("/timer", s.setTimer)
 			r.With(s.BoardParticipantContext).Delete("/timer", s.deleteTimer)
-
 			r.With(s.BoardModeratorContext).Put("/", s.updateBoard)
 			r.With(s.BoardModeratorContext).Delete("/", s.deleteBoard)
+			r.With(s.BoardModeratorContext).Put("/resetAllReadyStates", s.resetAllReadyStates)
```

Comment:
In the REST paradigm you rather manipulate resources than executing functions (e.g. in comparison to SOAP), what means to you could probably either create an endpoint in order to set the ready state of the users by `PUT` or you'll have an endpoint to maybe `DELETE` the ready state. Not sure what would be best but I hope you get the point.

My suggestions would be that you move the endpoint to `PUT /boards/<id>/participants/ready` or you could even include the readiness within the participants. Maybe we'll come up with other use cases where we want to change some flag of all participants.